documentation/modules/auxiliary/scanner/winrm/winrm_cmd.md

```diff
@@ -0,0 +1,34 @@
+## Description
+This module runs arbitrary Windows commands using the WinRM Service. It needs login credentials to do so.
```

Comment:
needs **valid** login credentials to do so.

I'd also ask what auth/group level they need to be.  Local admin, domain admin, guest account?